sdk/file_sdk/UnitTests/pfile_ut/pfile_tests.cpp

```diff
@@ -0,0 +1,55 @@
+#include <QString>
+#include <QtTest>
+#include <string>
+#include <QTStreamImpl.h>
+#include <PFileFileFormat/pfile_file_format.h>
+
+using mip::file::PFileFileFormat;
+
+class pfile_tests : public QObject
+{
+  Q_OBJECT
+
+public:
+  pfile_tests();
+
+private Q_SLOTS:
+  void GetTags_FileWithTag_ReturnCorrectTag();
+  void GetTags_FileWithoutTag_ReturnZeroTags();
+};
+
+pfile_tests::pfile_tests()
+{
+}
+
+void pfile_tests::GetTags_FileWithTag_ReturnCorrectTag()
+{
+  foreach( const QString &fileName, QDir(":files/taged_pfiles").entryList() )
+  {
+    QFile file(":files/taged_pfiles/" + fileName);
+    QSharedPointer<QDataStream> obj = QSharedPointer<QDataStream>(new QDataStream(&file));
```

Comment:
can't use this stream. this stream only works for files created with QDataStream